#! /bin/sh -e
## 07-manpage.dpatch by Luigi Gangitano <luigi@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixed wrong reference to /etc/squid/squid.conf

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/gangitano/debian/squid/squid-2.5.3/doc/squid.8 squid-2.5.3/doc/squid.8
--- /home/gangitano/debian/squid/squid-2.5.3/doc/squid.8	2000-12-18 22:54:18.000000000 +0100
+++ squid-2.5.3/doc/squid.8	2003-07-07 03:25:57.000000000 +0200
@@ -98,7 +98,7 @@
 .IP -Y
 Only return UDP_HIT or UDP_MISS_NOFETCH during fast reload.   
 .SH FILES
-.I /etc/squid/squid.conf
+.I /etc/squid.conf
 .RS
 The main configuration file.  You must initially make 
 changes to this file for 
